content/en/guides/hosting/hosting-options/self-managed/kubernetes-operator/_index.md

```diff
@@ -889,6 +893,322 @@ app:
+      exporters:
+        otlphttp:
+          endpoint: "${OTEL_ENDPOINT}"
+          headers:
+            api-key: "${OTEL_API_KEY}"
+          compression: gzip
+          timeout: 30s
+          sending_queue:
+            enabled: true
+            num_consumers: 10
+            queue_size: 1000
+          retry_on_failure:
+            enabled: true
+            initial_interval: 5s
+            max_interval: 30s
+            max_elapsed_time: 300s
+        prometheus:  # Keep local Prometheus for W&B console
+          endpoint: "0.0.0.0:9090"
+      receivers:
+        otlp:
+          protocols:
+            grpc:
+              endpoint: "0.0.0.0:4317"
+            http:
+              endpoint: "0.0.0.0:4318"
+        hostmetrics:
+          collection_interval: 10s
+          scrapers:
+            cpu: {}
+            disk: {}
+            filesystem: {}
+            load: {}
+            memory: {}
+            network: {}
+        k8s_cluster:
+          collection_interval: 10s
+        kubeletstats:
+          collection_interval: 20s
+          auth_type: serviceAccount
+          endpoint: "https://${env:K8S_NODE_NAME}:10250"
+          insecure_skip_verify: true
+        filelog:
+          include:
+            - /var/log/pods/*/*/*.log
+          start_at: end
+      service:
+        pipelines:
+          traces:
+            receivers: [otlp]
+            processors: [memory_limiter, k8sattributes, attributes, batch]
+            exporters: [otlphttp]
+          metrics:
+            receivers: [hostmetrics, k8s_cluster, kubeletstats, otlp]
+            processors: [memory_limiter, k8sattributes, attributes, batch]
+            exporters: [otlphttp, prometheus]
+          logs:
+            receivers: [filelog, otlp]
+            processors: [memory_limiter, batch, k8sattributes]
+            exporters: [otlphttp]
+    extraEnvs:
+      - name: OTEL_ENDPOINT
+        valueFrom:
+          secretKeyRef:
+            name: otel-secrets
+            key: endpoint
+      - name: OTEL_API_KEY
+        valueFrom:
+          secretKeyRef:
+            name: otel-secrets
+            key: api-key
+```
+
+#### Common processor types
+
+- **`batch`**: Groups telemetry data to reduce network overhead
+- **`memory_limiter`**: Prevents out-of-memory errors by limiting collector memory usage
+- **`k8sattributes`**: Enriches telemetry with Kubernetes metadata
+- **`attributes`**: Adds, updates, or removes attributes from telemetry data
+- **`resource`**: Adds or modifies resource attributes
+- **`filter`**: Excludes specific metrics, spans, or logs based on conditions
+- **`transform`**: Modifies telemetry data using a transformation language
+
+For more details on configuring OpenTelemetry collectors, refer to the [OpenTelemetry documentation](https://opentelemetry.io/docs/collector/configuration/) and the [OTLP HTTP exporter documentation](https://github.com/open-telemetry/opentelemetry-collector/tree/main/exporter/otlphttpexporter).
```

Comment:
Move this paragraph to the end of the top section of this before the Basic configuration section. In its place in this section, please link to more specific docs about processors.